### PR TITLE
Add .circleci/config.yml

### PR DESCRIPTION
**Pull request by koba04**

### CODE REVIEW

**Comment by koba04** [comment]
@ama-ch 
It seems to be working fine. Please review this 🙏

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# eslint-config-kintone [![npm version](https://badge.fury.io/js/eslint-config-kintone.svg)](https://badge.fury.io/js/eslint-config-kintone)
+# eslint-config-kintone
+
+[![npm version](https://badge.fury.io/js/eslint-config-kintone.svg)](https://badge.fury.io/js/eslint-config-kintone)
+[![CircleCI](https://circleci.com/gh/kintone/eslint-config-kintone.svg?style=svg)](https://circleci.com/gh/kintone/eslint-config-kintone)
```
Comment (teppeis) [inline README.md]: `?style=shield` is more suitable than `svg`

Comment (koba04) [reply README.md]: Thanks! I've fixed it.

**Comment by teppeis** [comment]
LGTM! https://circleci.com/gh/kintone/eslint-config-kintone

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+      - run: npm install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package-lock.json" }}
+
+      # run tests!
+      - run: npm test
```
Comment (ama-ch) [inline .circleci/config.yml]: How about running `npm run lint` here?

Comment (koba04) [reply .circleci/config.yml]: @ama-ch Thanks! That makes sense. I've added that.

**Comment by ama-ch** [comment]
LGTM! Thank you :octocat:

**Comment by koba04** [comment]
Thanks!

**Pull request merged**